web/Webhook.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from web import runner_m
+
+logger = logging.getLogger("runner_manager")
+
+
+class Webhook(object):
+    event: str
+    payload: dict
+
+    def __init__(self, payload: dict, event: str):
+        self.event = event
+        self.payload = payload
+
+    def __call__(self, *args, **kwargs):
+        # Check if we managed this event
+        if self.event not in [methode for methode in dir(self) if methode[:2] != "__"]:
+            logger.info(f"Event {self.event} not managed")
+        else:
+            logger.info(f'Get event: {event}')
+            getattr(self, self.event)(self.payload)
+
+    def workflow_run(self, payload):
+        pass
+
+    def workflow_job(self, payload):
+        status = {}
+        if payload['action'] == 'queued' or "self-hosted" in payload["workflow_job"]["labels"]:
+            return
```

Comment:
Help me out a bit here :sweat_smile: 
1. Shouldn't it be an `and` in the condition?
```suggestion
        if payload['action'] == 'queued' and "self-hosted" in payload["workflow_job"]["labels"]:
            return
```
2. Isn't this the moment where we can ask the runner manager to create an additional VM?